### ISSUE TITLE
nbconvert does not preserve markdown substitution.

### ISSUE THREAD

**Issue by yun881201** [jupyter/nbconvert#1887]
nbconvert does not preserve markdown substitution.

my format jupyter notebook file has markdown with substitution variable {{var}}.
using %store var=1 or 2 ... and nbconvert, I want to produce notebook containing markdown with variable var=1 or 2.
But nbconvert does not convert markdown with substitution variable {{var}} to markdown with substitution variable {{var}}.

**Comment by bollwyvl** [comment]
Yep. You'd have to write a kernel-specific preprocessor to support this.
